### PR TITLE
Bump SCF to CF v265.

### PR DESCRIPTION
**Pull request by andreas-kupries**
In a big surprise, it passed :cat: on the first run

### CODE REVIEW

File: bin/update-releases.sh [hunk 1]
```diff
@@ -56,25 +73,42 @@ do
...
+echo > ${GIT_ROOT}/_work/CF_RELEASE_VERSION_INFO "${CF_RELEASE_VERSION_INFO}"
+echo > ${GIT_ROOT}/_work/SUBMODULE_REFERENCES    ""
+
 get_submodule_ref () {
 	version_info=${1}
 	release=${2}
+
+	case $release in
+	    *-buildpack-*)
+		where=$(cd ${GIT_ROOT}/src/buildpacks/$release ; git remote -v | grep fetch | awk '{ print $2 }' | uniq)
```
Comment (alexjh) [inline bin/update-releases.sh]: You can use the `-C` flag in git to do this without `cd`ing.

`git -C "${GIT_ROOT}/src/buildpacks/$release" remote -v`

IMO a little more obvious, plus one less command being executed.

File: bin/update-releases.sh [hunk 2]
```diff
@@ -56,25 +73,42 @@ do
...
 
-GO_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" go-buildpack-release)
-BINARY_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" binary-buildpack-release)
-NODEJS_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" nodejs-buildpack-release)
-RUBY_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" ruby-buildpack-release)
-PHP_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" php-buildpack-release)
-PYTHON_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" python-buildpack-release)
-STATICFILE_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" staticfile-buildpack-release)
-JAVA_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" java-buildpack-release)
+# GO_BUILDPACK_RELEASE=$(get_submodule_ref "${CF_RELEASE_VERSION_INFO}" go-buildpack-release)
```
Comment (alexjh) [inline bin/update-releases.sh]: Why are these commented out?

File: container-host-files/etc/hcf/config/opinions.yml [hunk 3]
```diff
@@ -269,6 +268,7 @@ properties:
     url: https://this.is.unused
   metron_agent:
     deployment: hcf-deployment
+    health_port: 9999
```
Comment (alexjh) [inline container-host-files/etc/hcf/config/opinions.yml]: https://github.com/cloudfoundry/loggregator/commit/435af2536c7823448d6da5db96cac43a33295c85

I think this should be changed to `14824`, the default that upstream eventually agreed upon. The validation checks should flag this as unchanged from the default once we start using a release with this commit.

**Comment by alexjh** [comment]
@andreas-kupries Some merge conflicts appeared since you made this PR.

File: bin/update-releases.sh [hunk 4]
```diff
@@ -1,21 +1,38 @@
...
 
+# Save, for debugging
+mkdir -p ${GIT_ROOT}/_work
+echo "${VERSION_INFO}" > ${GIT_ROOT}/_work/VERSION_INFO
+
 CF_RELEASE=$(echo "${VERSION_INFO}" | jq -r .[\"cf-release-commit-sha\"])
 DIEGO_RELEASE=v$(echo "${VERSION_INFO}" | jq -r .[\"diego-release-version\"])
-CFLINUXFS2_ROOTFS_RELEASE=v$(echo "${VERSION_INFO}" | jq -r .[\"cflinuxfs2-rootfs-release-version\"])
+CFLINUXFS2_OLD_ROOTFS_RELEASE=v1.112.0
+CFLINUXFS2_ROOTFS_RELEASE=v$(echo "${VERSION_INFO}" | jq -r .[\"cflinuxfs2-release-version\"])
```
Comment (andreas-kupries) [inline bin/update-releases.sh]: Can do, will do.

File: bin/update-releases.sh (hunk 1 above)
Comment (andreas-kupries) [inline bin/update-releases.sh]: Sure, will do.

File: bin/update-releases.sh (hunk 2 above)
Comment (andreas-kupries) [inline bin/update-releases.sh]: The buildpacks are our own forks, they don't get bumped anymore, or rather, bumped independently of the scf core here.

File: container-host-files/etc/hcf/config/opinions.yml (hunk 3 above)
Comment (andreas-kupries) [inline container-host-files/etc/hcf/config/opinions.yml]: Didn't know about that. Will change.

**Comment by andreas-kupries** [comment]
I suspected conflicts when I noted that https://trello.com/c/EQCfgDNi/92-3-servicediscovery-svcclusterlocal-in-all-hostnames-needs-to-be-configurable was merged.

Will rebase.

File: bin/update-releases.sh (hunk 4 above)
Comment (andreas-kupries) [inline bin/update-releases.sh]: Done with commit 575040cb57bb2dea635062040a27ecdc96e0ba82

File: bin/update-releases.sh (hunk 1 above)
Comment (andreas-kupries) [inline bin/update-releases.sh]: Done with commit 64d2c35046d8f9778bc3e2e343ec430045b0f7b5

File: bin/update-releases.sh (hunk 2 above)
Comment (andreas-kupries) [inline bin/update-releases.sh]: Added explanation with commit 64d2c35046d8f9778bc3e2e343ec430045b0f7b5

File: container-host-files/etc/hcf/config/opinions.yml (hunk 3 above)
Comment (andreas-kupries) [inline container-host-files/etc/hcf/config/opinions.yml]: Done with commit 0224f9502a7076069cbd41a951f188591683b981

**Comment by andreas-kupries** [comment]
Rebased, fixed a few leftover `HCP_SERVICE_DOMAIN_SUFFIX` in my branch.
Passes :smoking: and :cat:

**Comment by alexjh** [comment]
LGTM

**Pull request merged**